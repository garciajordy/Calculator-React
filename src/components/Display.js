import React from 'react';

const Display = () => (
  <h1>
    0
  </h1>
);

export default Display;
