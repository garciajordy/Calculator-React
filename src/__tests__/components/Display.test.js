import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Display from '../../components/Display';

afterEach(cleanup);

it('should take a snapshot', () => {
  const { asFragment } = render(<Display result="some value" />);
  expect(asFragment(<Display result="some value" />)).toMatchSnapshot();
});
