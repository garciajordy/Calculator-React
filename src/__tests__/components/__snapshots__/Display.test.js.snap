// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <h1
    class="display"
  >
    some value
  </h1>
</DocumentFragment>
`;
